### PR TITLE
chore(zh-cn): translate for getTargetRanges()

### PR DESCRIPTION
**Pull request by fuchunhui**
### Description
1. create new content.

### CODE REVIEW

File: files/zh-cn/web/api/inputevent/gettargetranges/index.md [hunk 1]
```diff
@@ -0,0 +1,98 @@
+---
+title: InputEvent：getTargetRanges() 方法
+slug: Web/API/InputEvent/getTargetRanges
+l10n:
+  sourceCommit: 72ca3d725e3e56b613de3ac9727bd0d6d619c38a
+---
+
+{{APIRef("UI Events")}}
+
+{{domxref("InputEvent")}} 接口的 **`getTargetRanges()`** 方法返回一个 {{domxref("StaticRange")}} 对象数组，如果不取消输入事件，该数组受到 DOM 更改的影响。
```
Comment (JasonLamv-t) [inline files/zh-cn/web/api/inputevent/gettargetranges/index.md:10]: Good catch! Please create a PR to update to fix the macro-refer to "Static Range"

File: files/zh-cn/web/api/inputevent/gettargetranges/index.md [hunk 2]
```diff
@@ -0,0 +1,98 @@
...
+
+这允许 web 应用程序在浏览器修改 DOM 树之前重写文本编辑行为，提供对输入事件更多的控制以提高性能。
```
Comment (JasonLamv-t) [inline files/zh-cn/web/api/inputevent/gettargetranges/index.md:12]:
```suggestion
这允许 web 应用程序在浏览器修改 DOM 树之前重写文本编辑行为，并提供更多控制输入事件的能力，以提高性能。
```

File: files/zh-cn/web/api/inputevent/gettargetranges/index.md [hunk 3]
```diff
@@ -0,0 +1,98 @@
...
+  <thead>
+    <tr>
+      <th>inputType</th>
+      <th>编辑宿主</th>
+      <th><code>getTargetRanges()</code>的响应</th>
+    </tr>
+  </thead>
+  <tbody>
+    <tr>
+      <td><code>"historyUndo"</code> or <code>"historyRedo"</code></td>
```
Comment (JasonLamv-t) [inline files/zh-cn/web/api/inputevent/gettargetranges/index.md:26]:
```suggestion
      <td><code>"historyUndo"</code> 或 <code>"historyRedo"</code></td>
```

**Review by JasonLamv-t — COMMENTED** [review]
Well done! Just some nits.

File: files/zh-cn/web/api/inputevent/gettargetranges/index.md (hunk 1 above)
Comment (fuchunhui) [reply files/zh-cn/web/api/inputevent/gettargetranges/index.md:10]: https://github.com/mdn/content/pull/32910

**Review by JasonLamv-t — APPROVED** [review]
Well done! Thank you! @fuchunhui

**Pull request merged**